tests/storage/lvm_test.py

```diff
@@ -1179,6 +1191,32 @@ def test_vg_get_all_hit_no_stale(fake_devices):
     assert len(vgs) == num_vgs
 
 
+def test_vg_remove_by_uuid(monkeypatch, fake_devices):
```

Comment:
We should test this using real vg, not fakes.

You can add a simple test like:
https://github.com/oVirt/vdsm/blob/013692737dc9dd89b977cb8a91e479c8757fd0db/tests/storage/lvm_test.py#L921